pkg/kubelet/server/portforward/websocket.go

```diff
@@ -43,25 +44,33 @@ const (
 // options contains details about which streams are required for
 // port forwarding.
 type v4Options struct {
-	port uint16
+	ports []uint16
 }
 
 // newOptions creates a new options from the Request.
 func newV4Options(req *http.Request) (*v4Options, error) {
-	portString := req.FormValue(api.PortHeader)
-	if len(portString) == 0 {
+	portStrings := req.URL.Query()[api.PortHeader]
+	if len(portStrings) == 0 {
 		return nil, fmt.Errorf("%q is required", api.PortHeader)
 	}
-	port, err := strconv.ParseUint(portString, 10, 16)
-	if err != nil {
-		return nil, fmt.Errorf("unable to parse %q as a port: %v", portString, err)
-	}
-	if port < 1 {
-		return nil, fmt.Errorf("port %q must be > 0", portString)
+
+	ports := make([]uint16, 0, len(portStrings))
+	for _, portString := range portStrings {
+		if len(portString) == 0 {
+			return nil, fmt.Errorf("%q is cannot be empty", api.PortHeader)
+		}
+		port, err := strconv.ParseUint(portString, 10, 16)
```

Comment:
Compare https://tools.ietf.org/html/rfc2616#section-4.2. Comma separated lists must be supported as well.